frontend_tests/node_tests/stream_data.js

```diff
@@ -599,3 +605,10 @@ zrequire('marked', 'third/marked/lib/marked');
     assert.equal(antarctica_sub.color, '#76ce90');
 }());
 
+(function test_invite_streams() {
+    var expected_list = _.union(
```

Comment:
This is the definition of `invite_streams()`, so this is not going to make any useful comparison. I think it would be better if you just write out the value of `expected_list` here explicitly.

Reply:
Doing it right now.